src/components/InputDropdown/index.tsx

```diff
@@ -150,7 +150,6 @@ export default function InputDropdown({
         onChange={handleChange}
         onInputChange={handleInputChange}
       />
-      {error && <ErrorText>{error}</ErrorText>}
```

Comment:
are we removing this because we're handling error messages with zod?

Reply:
this is mostly because `error` is a boolean, so the error text that shows up displays nothing at all, and only takes up space

Reply:
plus, in the usage that we have with `InputDropdown`, they've always been accompanied by `FormMessage` - which shows the error text for us when there is an error, so this becomes redundant